ci-regression.sh

```diff
@@ -27,7 +27,7 @@ if [[ -n "$TRAVIS_PULL_REQUEST_SHA" ]] && [[ -n "$TRAVIS_BRANCH" ]]; then
   git stash
   git checkout "$TRAVIS_BRANCH"
 elif [[ -n "$TRAVIS_COMMIT_RANGE" ]]; then
-  prev=${TRAVIS_COMMIT_RANGE%%.*}~1
+  prev=${TRAVIS_COMMIT_RANGE%%.*}
```

Comment:
You are now checking out the first commit in this PR. You are no longer comparing with master. You're comparing with your own first commit.

Reply:
We do need to get the Travis docs fixed because what I am seeing is true.

Look at the last job on master that was the result of HitCoder's pr being merged:
https://travis-ci.org/enigma-dev/enigma-dev/jobs/414325369

>This appears to be a Travis push integration run; checking out '42b7d98efe99' for the comparison.
Previous HEAD position was e1e3e0a0... Fix surface_create signature in sources other than GL3 (#1353)

Here's the commit that did the merge:
https://github.com/enigma-dev/enigma-dev/commit/e1e3e0a080d56f5c9950afb5c24b08f29937b34b

The parent of e1e3e0a080d56f5c9950afb5c24b08f29937b34b is 42b7d98efe9930acb3ade71205b3778108892730 which is exactly what the job claims to have checked out.

So the Travis docs need fixed 😛

Reply:
It seems someone already tried reporting this to the Travis team but they ignored it:
https://github.com/travis-ci/travis-ci/issues/4596

Their bot automatically closed the issue as stale.